sdk_contrib/fetch/lib/fetch_p.js

```diff
@@ -116,7 +120,7 @@ const enableCapture = function enableCapture(baseFetchFunction, requestClass, do
       const requestClone = request.clone();
       let response;
       try {
-        response = await baseFetchFunction(requestClone);
+        response = await baseFetchFunction(requestClone, fetchOptions);
```

Comment:
Could this be changed to `baseFetchFunction(...args);`? This way the inputs to the fetch function are as what was originally intended as if there were no instrumentation at all. There would be no need to create `fetchOptions` either.